guides/doc-Configuring_Smart_Proxies_with_a_Load_Balancer/topics/Configuring_Capsule_Server_with_Custom_SSL_Certificates_and_without_Puppet_for_Load_Balancing.adoc

```diff
@@ -0,0 +1,168 @@
+[id='configuring-capsule-server-with-custom-ssl-certificates-for-load-balancing-without-puppet']
+
+= Configuring {SmartProxyServer} with Custom SSL Certificates for Load Balancing without Puppet
```

Comment:
Ditto, maybe that's what you want.

Reply:
Personally I prefer not to have the extra line, but I know some people preferred it... I think maybe to make up for a quirk the Customer Portal had? I dunno if it still has that quirk, but in any case I'd planned to delete the extra line in a later mod docs PR. I'm happy to do it now, though. :+1: